mirrord/agent/src/steal/connection.rs

```diff
@@ -389,7 +389,15 @@ impl TcpConnectionStealer {
 
     /// Initialize iptables member, which creates an iptables chain for our rules.
     fn init_iptables(&mut self) -> Result<()> {
-        self.iptables = Some(SafeIpTables::new(iptables::new(false).unwrap())?);
+        let flush_connections = std::env::var("MIRRORD_AGENT_STEALER_FLUSH_CONNECTIONS")
```

Comment:
I had a wrong assumption that we have a file with all env settings for agent in one place and wanted you to add it there but now i see there's no such file so we should have a todo on that because having env spread around the code is evil :)